tests/PactNet.Tests/Matchers/ArrayContainsMatcherTests.cs

```diff
@@ -0,0 +1,252 @@
+using System;
+using System.Collections.Generic;
+using System.Net;
+using System.Net.Http;
+using System.Text.Json;
+using System.Threading.Tasks;
+using FluentAssertions;
+using PactNet.Matchers;
+using Xunit;
+
+namespace PactNet.Tests.Matchers;
+
+public class ArrayContainsMatcherTests
+{
+    private readonly IPactBuilderV4 pactBuilder;
+
+    public ArrayContainsMatcherTests()
+    {
+        pactBuilder = Pact.V4("Some Consumer", "Some Producer")
+            .WithHttpInteractions();
+    }
+
+    [Fact]
+    public async Task ConsumerTests_Simple_StringList()
+    {
+        // Arrange
+        this.pactBuilder
+            .UponReceiving("A request for strings")
+            .WithRequest(HttpMethod.Get, "/api/list/string")
+            .WillRespond()
+            .WithStatus(HttpStatusCode.OK)
+            .WithJsonBody(Match.ArrayContains([
+                "Thing 1",
+                "Thing 2"
+            ]));
+
+        await this.pactBuilder.VerifyAsync(async ctx =>
+        {
+            using var service = Service(ctx.MockServerUri);
+
+            // Act
+            var strings = await service.GetStringList();
+
+            // Assert
+            strings.Should().Contain("Thing 1").And.Contain("Thing 2");
```

Comment:
suggestion: Use `Should().Equal(["Thing 1", "Thing 2"]}` to be more explicit that it should contain all of the expected, and only the expected